### DISCUSSION TITLE
Can github get a spam prevention by bots/compromised accounts?

### DISCUSSION THREAD

**Discussion by DuckersMcQuack** [community/community#61465] [Discussions]
### Select Topic Area

Product Feedback

### Body

I have seen a certain developer for a program who 2 times now has been spammed by bots/hijacked accounts and spams the issues section with the same badly executed phishing over like 15 issues. Can we get a security feature where the automated system has these features or something similar?

Spam once, it's posted
Spam again with the same post, it's erroring saying it's already been posted
Spam a third time, get a timeout
Spam a 4'th time attempt after, the security then disables that account's posting privileges, and you're blocked from posting on that repo, and any other repos with the same copypasta.

And a report system for each such spam where the similarity of the message gets auto reported/message deleted/muted for the future spams on the same/other repos.

**Comment by la-cc** [comment]
I've observed a situation where a certain developer's program has been targeted twice now by bot accounts or hijacked accounts that flood the issues section with poorly executed phishing attempts, spread over multiple issues. To combat this, I suggest the introduction of a security feature in the automated system with the following, or similar, capabilities:

1. On the first instance of spam, the post is allowed.
2. If the same post is attempted again, an error message indicates that it's already been posted.
3. On a third attempt, the offending account receives a timeout.
4. After a fourth attempt, the security system disables that account's posting privileges. The account is blocked from posting on that repository and any other repositories with the same copied content.
